lwc/src/main/scala/fs2conversion.scala

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2014–2018 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package spinoco.scalaz.stream
+
+import slamdata.Predef.{Stream => _, _}
+import scala.Predef.identity
+
+import fs2._
+
+import scalaz.concurrent.{Actor, Task}
+import scalaz.stream.Process
+import scala.language.higherKinds
+import scalaz.{-\/, \/, \/-}
+import shims._
+
+// ALL of the code below has been converted from fs2 0.9 to 0.10
+// from a gist of code originally written by Pavel Chlupacek (@pchlupacek on github)
+// and shared with the community as https://gist.github.com/pchlupacek/989a2801036a9441da252726a1b4972d
```

Comment:
Any news if this will be published at any point for us to consume?

Reply:
Also, why do we need to interop with `scalaz.stream` at all?

Reply:
:+1: for the first version of the `S3` connector this is fine. 

@djspiewak  before we put the `S3` connector in the hands of any customers, I'd like to get quasar updated from `scalaz-stream` and old `fs2` versions to the latest `fs2`. Else we have to go through logic like this below, which I don't trust at all.